src/BaGetter/Startup.cs

```diff
@@ -77,6 +90,11 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             app.UseStatusCodePages();
         }
 
+        // activate HTTP logging option
+        // The default severity on the Console is currently "Warning".
+        // So, the logging is not visible by default but can be easily activated using environment variable.
```

Comment:
Won't warnings and errors still get logged by default? So "most of the logging..." would be more accurate.